### DISCUSSION TITLE
libpag支不支持Electron-Vue框架项目呢？

### DISCUSSION THREAD

**Discussion by yufengvac** [Tencent/libpag#2328] [General] [answered]
我有个 Electron-vue项目，想在这个项目里面加载PAG并显示，请问下是否支持Electron桌面端项目？

**Comment by kevingpqi123** [comment, marked as answer]
不支持，由于我们自身人力有限以及没有此类的业务场景，这部分是没有做处理的，pag 源码是开源的，欢迎大家来完善这部分内容，短时间内我们这边应该没有人力来支持该框架
